application/controllers/QuestionAdministrationController.php

```diff
@@ -1166,6 +1166,18 @@ public function actionImport()
         $iSurveyID = (int) App()->request->getPost('sid', 0);
         $gid = (int) App()->request->getPost('gid', 0);
 
+        if (!Permission::model()->hasSurveyPermission($surveyid, 'surveycontent', 'import')) {
+            App()->user->setFlash('error', gT("Access denied"));
+            $redirect = Yii::app()->createUrl(
```

Comment:
Bad redirect ?

This goes to a second redirect : https://github.com/LimeSurvey/LimeSurvey/blob/f9316bf0a2d47feb8fd5bc5188418d80d46487c4/application/controllers/QuestionAdministrationController.php#L1132